### ISSUE TITLE
[discussion] Why is this plugin deprecated?

### ISSUE THREAD

**Issue by markdegrootnl** [apache/cordova-plugin-file-transfer#266]
>With the new features introduced in XMLHttpRequest, this plugin is not needed any more. Migrating from this plugin to using the new features of XMLHttpRequest, is explained in this Cordova blog post.
-- https://github.com/apache/cordova-plugin-file-transfer#deprecated

Sure downloading data as a blob or arraybuffer and saving it using [cordova-plugin-file](https://github.com/apache/cordova-plugin-file) works perfectly fine, however there is one big caveat.

I've run several tests downloading a 70MB file and two things stand out:

1. Downloading the file and saving it to the device' storage is significantly slower using XHR and the File plugin compared to the FileTransfer plugin. I'll elaborate on this later on.
2. When trying to improve the download speed, I've tried downloading the file as a Blob and a ArrayBuffer. Blob works, however slow. Downloading as ArrayBufer results in a memory exhaustion with just a 70 MB download.

Now for some cold hard facts:
- Downloading my 70MB test file using the FileTransfer plugin takes about 35 seconds. 
- Downloading using XHR takes about 30 seconds, however the file still needs to be saved to storage using the File plugin. This takes another 50 seconds resulting in a total download time of 80 seconds! 

The main reason for this massive difference in performance lies in the fact that the FileTransfer plugin executes the entire process in native code and probably uses streams to download and save the file in one single step. The proposed alternative using XHR downloads the entire file in javascript first and sends the entire data Blob to the native File plugin to save the data to storage. 

So in conclusion: Why is this plugin deprecated if the proposed alternative is notably slower?

**Comment by timbru31** [comment]
Please see https://github.com/apache/cordova-plugin-file-transfer/issues/258#issuecomment-662907931 - your input on the mailing list would be appreciated.

**Comment by markdegrootnl** [comment]
Hi @timbru31 I've added my input to the mailing list but is does not appear on the page. Does this take a while?

**Comment by timbru31** [comment]
Yes, might take a bit. I’ve already got an email. Thanks!

**Comment by markdegrootnl** [comment]
Hi @timbru31 My input still hasn't been added to the mailing list page. Any idea why?

**Comment by dpa99c** [comment]
Note: after a PMC vote, this plugin has been undeprecated

**Comment by ssledorze** [comment]
PMC ?

**Comment by dpa99c** [comment]
Apache [Project Management Commitee](https://www.apache.org/dev/pmc.html) - in the case the [Cordova PMC](https://projects.apache.org/committee.html?cordova)
